### ISSUE TITLE
Remove all the MvxBinding junk from the app

### ISSUE THREAD

**Issue by heytherewill** [toggl/mobileapp#3869]
Doable only once all views are RxFactored. We need to remove:

- [x] The `Bindings` folder from both `Daneel` and `Giskard`
- [x] The `Combiners` folder from `Foundation`, `Daneel` and `Giskard`
- [x] The `Converters` folder from `Foundation`, `Daneel` and `Giskard`
- [x] The `SetupTargetFactories.tt` from `Daneel` and `Giskard`
- [x] Everything that's no longer needed from `LinkerPleaseInclude.cs`. Let the linker strip all of that away!
